DDDEastAnglia/Areas/Admin/Views/Sponsor/Delete.cshtml

```diff
@@ -0,0 +1,18 @@
+﻿@model DDDEastAnglia.Models.Sponsor
+
+@{
+    ViewBag.Title = string.Format("Delete {0}?", Model.Name);
+}
+
+<h2>@ViewBag.Title</h2>
+
+<p>Deleting a sponsor cannot be undone. Are you <em>really</em> sure you want to delete this sponsor?</p>
```

Comment:
This feels like it should be reusable beyond this scenario (and don't we have other deletion confirmations elsewhere in the system?).